google/resource_compute_instance_test.go

```diff
@@ -10,6 +10,7 @@ import (
 	"github.com/hashicorp/terraform/helper/acctest"
```

Comment:
Same comments about the tests as I had for the instance template one, cases 3 and 4 seem unnecessary and `Skip` is redundant with the `_basic` tests.